src/main/java/essenmakanan/command/UseIngredientCommand.java

```diff
@@ -0,0 +1,54 @@
+package essenmakanan.command;
+
+import essenmakanan.exception.EssenFormatException;
+import essenmakanan.ingredient.Ingredient;
+import essenmakanan.ingredient.IngredientList;
+import essenmakanan.parser.IngredientParser;
+import essenmakanan.ui.Ui;
+
+public class UseIngredientCommand extends Command {
+    private String toUse;
+    private IngredientList ingredients;
+
+    public UseIngredientCommand(IngredientList ingredients, String toUse) {
+        super();
+        this.toUse = toUse;
+        this.ingredients = ingredients;
+    }
+
+    @Override
+    public void executeCommand() {
+        String[] allIngredients = toUse.split("i/");
+
+        for (String ingredient : allIngredients) {
+            if (ingredient.isEmpty()) {
+                continue;
+            }
+
+            updateIngredientQuantity(ingredient);
```

Comment:
Would it be better to do if(!ingredient.isEmpty()){...} so that everything is handled within the if statement?